doc/auto_techsupport_and_coredump_mgmt.md

```diff
@@ -374,3 +375,8 @@ Enhance the existing techsupport sonic-mgmt test with the following cases.
 |  3   | Check if the global rate-& & per-process rate-limit-interval is working as expected                                                     |
 |  4   | Check if the core-dump cleanup is working as expected                                                                                   |
 
+## 8. SONiC-to-SONiC Upgrade Considerations
+
+The default config required for auto_techsupport is present in the init_cfg.json. Therefore, when a clean installation of SONiC is performed, the configuration is found in the config DB and the feature is active. 
+
+However, in the case of SONiC-SONiC upgrade, the previous config_db.json is migrated and init_cfg.json is not involved. In that case, it is the responsibility of the admin to provide the config, if the admin wants to leverage this feature. 

```

Comment:
>the admin to provide the config [](http://example.com/codeflow?start=162&length=31)

Could you give the instruction to provide the config in case of SONiC-SONiC upgrade?

Reply:
Added